### PR TITLE
Add remark command

### PR DESCRIPTION
**Pull request by officialchengyud**
tP week 6 requirement

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -21,11 +21,7 @@
 import seedu.address.logic.Messages;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Person;
-import seedu.address.model.person.Phone;
+import seedu.address.model.person.*;
```
Comment (alfaloo) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:24]: Should not have wildcard import

File: src/main/java/seedu/address/storage/JsonAdaptedPerson.java [hunk 2]
```diff
@@ -10,11 +10,7 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 import seedu.address.commons.exceptions.IllegalValueException;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Person;
-import seedu.address.model.person.Phone;
+import seedu.address.model.person.*;
```
Comment (alfaloo) [inline src/main/java/seedu/address/storage/JsonAdaptedPerson.java:13]: Again, wildcard importing should be avoided

File: src/test/java/seedu/address/logic/commands/RemarkCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import org.junit.jupiter.api.Test;
+import seedu.address.model.AddressBook;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Remark;
+import seedu.address.testutil.PersonBuilder;
+
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (alfaloo) [inline src/test/java/seedu/address/logic/commands/RemarkCommandTest.java:12]: Again, wild card imports

**Comment by Kappaccinoh** [comment]
ethans comments are brutal man 4 strikes and you're out @officialchengyud

**Pull request closed without merging**